website/snippets/_new-sl-setup.md

```diff
@@ -63,20 +72,36 @@ Note that:
 
 To add multiple credentials and map them to service tokens:
 
-1. After configuring your environment, on the **Credentials & service tokens** page click the **Add Semantic Layer credential** button to configure the credential for your data platform. 
-2. On the **Create New Semantic Layer Credential** page, you can create multiple credentials and map them to a service token.
-3. In the **Add credentials** section, fill in the data platform's credential fields. We recommend using “read-only” credentials.
+1. After configuring your environment, on the **Credentials & service tokens** page, click the **Add Semantic Layer credential** button to create multiple credentials and map them to a service token.
+2. In the **1. Add credentials** section, fill in the data platform's credential fields. We recommend using “read-only” credentials.
 <Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-add-credential.jpg" width="55%" title="Add credentials and map them to a service token. " />
-4. In the **Map new service token** section, map a service token to the credential you configured in the previous step. dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
-   - To add another service token, click **Add service token** under the **Linked service tokens** section.
+
+3. In the **2. Map new service token** section, map a service token to the credential you configured in the previous step. dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
+
+4. To add another service token during configuration, click **Add permission**. 
+   - (To add another service token to an existing Semantic Layer configuration, click **Add service token** under the **Linked service tokens** section in the **Semantic Layer Configuration Details** page.)
```

Comment:
I think this change isn't right. :/

So, the "add permission" button adds permissions groups like the pictured "Semantic Layer Only" and "Metadata Only" permission groups.  You can add more permissions (there are so many, from github admin to billing...) if you want to re-use this token for other types of dbt (non-SL) tasks.  We don't recommend that, but it's also not necessarily wrong to do so. 

This button does not add new service tokens.

-----

The bulleted point here below 4 is correct though, although it might be better to say something like "you can link more service tokens to this later in the **Semantic Layer Configuration Details** page" and then explain it as a different number since it requires leaving this workflow.

Reply:
oh great shout! yes i confused that, thank you for flagging that!!